torch/csrc/cuda/Stream.h

```diff
@@ -11,8 +12,7 @@ struct THCPStream {
   // the hook for placement-new'ing/delete'ing it; simpler to just
```

Comment:
Doesn't the comment above apply to `CUDAStream` as well? It is not a POD either.

Reply:
This code comment is no longer relevant (and should be deleted)